Bug: GPX export crashes on multi-track export

Reported via GooglePlay Store.

Somehow one can trigger a multi-track export to GPX.
And that is not supported by the GPXTrackExporter.

```
java.lang.RuntimeException: 
  at android.os.AsyncTask$4.done (AsyncTask.java:399)
  at java.util.concurrent.FutureTask.finishCompletion (FutureTask.java:383)
  at java.util.concurrent.FutureTask.setException (FutureTask.java:252)
  at java.util.concurrent.FutureTask.run (FutureTask.java:271)
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1167)
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:641)
  at java.lang.Thread.run (Thread.java:919)
Caused by: java.lang.UnsupportedOperationException: 
  at de.dennisguse.opentracks.io.file.exporter.GPXTrackExporter.writeTrack (GPXTrackExporter.java:108)
  at de.dennisguse.opentracks.content.provider.ShareContentProvider.lambda$openFile$1 (ShareContentProvider.java:228)
  at de.dennisguse.opentracks.content.provider.-$$Lambda$ShareContentProvider$UUYLBGQZWBfxU6qi91pkc6uXicw.writeDataToPipe (Unknown Source:11)
  at android.content.ContentProvider$1.doInBackground (ContentProvider.java:2033)
  at android.os.AsyncTask$3.call (AsyncTask.java:378)
  at java.util.concurrent.FutureTask.run (FutureTask.java:266)
```

--- a/src/main/java/de/dennisguse/opentracks/io/file/exporter/GPXTrackExporter.java
+++ b/src/main/java/de/dennisguse/opentracks/io/file/exporter/GPXTrackExporter.java
@@ -34,9 +34,10 @@ import de.dennisguse.opentracks.content.provider.TrackPointIterator;
 import de.dennisguse.opentracks.util.StringUtils;
 
 /**
- * Convert {@link Track} incl. {@link Marker} and {@link TrackPoint} to KML.
+ * Convert {@link Track} incl. {@link Marker} and {@link TrackPoint} to GPX.
  * NOTE:
  * * does not export {@link TrackPoint} without a latitude/longitude (not supported by GPX 1.1).
+ * * cannot export multiple {@link Track}s - all {@link TrackPoint}s are exported as if they would belong to the first track.
  *
  * @author Sandor Dornbush
  * @author Rodrigo Damazio
@@ -85,13 +86,21 @@ public class GPXTrackExporter implements TrackExporter {
 
     @Override
     public boolean writeTrack(Track track, @NonNull OutputStream outputStream) {
+        return writeTrack(new Track[]{track}, outputStream);
+    }
+
+    @Override
+    public boolean writeTrack(Track[] tracks, @NonNull OutputStream outputStream) {
+        Track mainTrack = tracks[0];
         try {
             prepare(outputStream);
-            writeHeader(track);
+            writeHeader(mainTrack);
 
-            writeMarkers(track);
+            writeMarkers(mainTrack);
 
-            writeLocations(track);
+            for (Track track : tracks) {
+                writeLocations(track);
+            }
 
             writeFooter();
             close();
@@ -101,11 +110,6 @@ public class GPXTrackExporter implements TrackExporter {
             Log.e(TAG, "Thread interrupted", e);
             return false;
         }
-    }
-
-    @Override
-    public boolean writeTrack(Track[] tracks, @NonNull OutputStream outputStream) {
-        throw new UnsupportedOperationException();
     }
 
     private void writeLocations(Track track) throws InterruptedException {

